lnrpc/routerrpc/router.proto

```diff
@@ -60,15 +60,15 @@ message SendPaymentRequest {
     */
     int64 fee_limit_sat = 7;
 
-     /**
-    The maximum number of millisatoshis that will be paid as a fee of the
-    payment. If this field is left to the default value of 0, only zero-fee
-    routes will be considered. This usually means single hop routes connecting
-    directly to the destination. To send the payment without a fee limit, use
-    max int here.
-    
-    The fields fee_limit_sat and fee_limit_msat are mutually exclusive.
-    */
+    /**
```

Comment:
nit: looks like the formatter failed to catch the extra space here.

Reply:
That commit is from the dependent PR where the formatter was not yet introduced. I noticed this and fixed it in the previous PR because the formatter found it.

Reply:
Sorry missed the parent PR.